docs/faq.rst

```diff
@@ -29,7 +29,8 @@ FAQ
     Double machine learning is an ATE estimation technique, pioneered by
     `Chernozhukov et al. (2016) <https://arxiv.org/abs/1608.00060>`_.
     It is 'double' in the sense that it relies on two preliminary models: one for the probability of
-    receiving treatment given covariates (the propensity score), and one for the outcome given treatment and covariates.
+    receiving treatment given covariates (the propensity score), and one for the outcome covariates and
+    optionally the treatment.
```

Comment:
This makes now a lot more sense to me with the clarification of the terms nuisance, propensity score, and treatment you gave before! I would write here though: and optionally the treatment, which corresponds to the last-step (regression) procedure in DML.

Reply:
I'm not following what you mean here, the two preliminary models explained here are the nuisance models (outcome and propensity), the last step procedure in DML comes after this two models afaict.

Reply:
I thought that by "optionally the treatment" you meant the OLS regression that is carried out as a last stage in DML and thought one could write this a bit clearer. What did you mean by "optionally the treatment"?

Reply:
[1856b68](https://github.com/Quantco/metalearners/pull/21/commits/1856b685415c9903edfba46fa7a230a206e39886)